satellite/orders/endpoint.go

```diff
@@ -41,8 +41,8 @@ type DB interface {
 
 	// UpdateStoragenodeBandwidthAllocation updates 'allocated' bandwidth for given storage node
 	UpdateStoragenodeBandwidthAllocation(ctx context.Context, storageNode storj.NodeID, action pb.PieceAction, amount int64) error
-	// UpdateStoragenodeBandwidthSettle updates 'settled' bandwidth for given storage node
-	UpdateStoragenodeBandwidthSettle(ctx context.Context, storageNode storj.NodeID, action pb.PieceAction, amount int64) error
+	// UpdateStoragenodeBandwidthSettleWithCustomDate updates 'settled' bandwidth for given storage node for the given intervalStart time
+	UpdateStoragenodeBandwidthSettle(ctx context.Context, storageNode storj.NodeID, action pb.PieceAction, amount int64, intervalStart time.Time) error
```

Comment:
I feel like it looks a bit odd that this method takes the intervalStart arg, but none of the others do. Should we change the others to be consistent even though they don't actually need it?

Reply:
@egonelbre what do you think?